src/objects/conf/base.py

```diff
@@ -388,13 +388,23 @@
         **SENTRY_CONFIG, integrations=SENTRY_SDK_INTEGRATIONS, send_default_pii=True
     )
 
+#
 # Elastic APM
-
+#
+ELASTIC_APM_SERVER_URL = os.getenv("ELASTIC_APM_SERVER_URL", None)
 ELASTIC_APM = {
-    "SERVICE_NAME": "objects",
+    "SERVICE_NAME": os.getenv("ELASTIC_APM_SERVICE_NAME", "Objects API"),
     "SECRET_TOKEN": os.getenv("ELASTIC_APM_SECRET_TOKEN", "default"),
-    "SERVER_URL": os.getenv("ELASTIC_APM_SERVER_URL", "http://example.com"),
+    "SERVER_URL": ELASTIC_APM_SERVER_URL,
 }
+if not ELASTIC_APM_SERVER_URL:
+    ELASTIC_APM["ENABLED"] = False
+    ELASTIC_APM["SERVER_URL"] = "http://localhost:8200"
```

Comment:
Why not `None` here?

Reply:
Whoops, I was a bit to eager copy pasting the setup from Open Forms https://github.com/open-formulieren/open-forms/commit/1d70a6d3d2037d1aa5b27b0190843d0f4f076007#diff-29ee1cea9a50ac782bbeedda8cd63a566bbe1dd6ed669143850f56482402d3a6R484

I'll change it to None